### PR TITLE
Remove all MShock derived tilesets.

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
SUMMARY: Content "Remove all MShock derived tilesets."

#### Purpose of change
It was brought to my attention that the DeadPeople tileset contains a copy of a tile from the Minecraft HarvestCraft mod.
https://harvestcraftmod.fandom.com/wiki/Epic_Bacon
A copy of this tile is used for the "Jerky" tile.
Upon further research, it turns out this tile is also used in the MShock and Chesthole tilesets.

#### Describe the solution
Remove all of the affected tilesets and their dependent mods.
Change the default tileset to retrodays.

#### Additional context
In addition to this particular tile being a license violation, it casts suspicion on the sourcing of the remainder of the tiles, I can't in good conscience include any of these tilesets without them being cleaned up and the provenance of the tiles being established.

### CODE REVIEW

**Comment by 12347654** [comment]
Fuck you fat scum

**Comment by 12347654** [comment]
fuck you fat scum again :3

**Comment by Firestorm01X2** [comment]
> A copy of this tile is used for the "Jerky" tile.

So. Instead of blacking out on tile whole tileset gone away. Pretty rushed solution.
And it is huge step back in development.

I heard that Dead People tileset author deleted his sources because of that.

It is really bad.

**Comment by Inglonias** [comment]
> A copy of this tile is used for the "Jerky" tile.
> 
> So. Instead of blacking out on tile whole tileset gone away. Pretty rushed solution.
> And it is huge step back in development.

The sad truth is that it could very well be just that one tile, but if it isn't, and the license holders find out, that lands the entire project in a whole heap of trouble. The only responsible thing to do is to take it down. If you're willing and able to prove that the rest of the tileset is fine, do so and it may come back. Unless and until that happens, the risk is simply too high to keep those tilesets in the repo.

> I heard that Dead People tileset author deleted his sources because of that.
> It is really bad.
That's not exactly something you can blame Kevin for. SDG didn't make that tile, and I personally suspect that their original work is exactly that - original. But the tile in question was found in the original MShock tileset, and AFAIK, the author of that isn't around anymore, so we can't ask them what else they may have used inappropriately. For all I know, it wasn't intentional. They may have just used stuff and didn't know it could cause problems.

The bottom line is that anyone can still use affected tilesets, but they can't be included in this repo without potential trouble, especially now that we know about the problem.

**Comment by Firestorm01X2** [comment]
> That can still happen, and when that's done, that's when the tile sets can come back up.

Now I've really dounbt that SDG will work on that. And I can undererstnad him. And since authors of ther tilesets gone - there is problem.

**Comment by anothersimulacrum** [comment]
SDG'S removal of the source was unrelated

https://discourse.cataclysmdda.org/t/abandoned-32x32-msx-dead-people-tileset/18775/370

**Comment by Firestorm01X2** [comment]
Still I have suspicion that this tile was around for a  long time.
So next step- removing built versions and stable versions?

**Comment by ZhilkinSerg** [comment]
> 
> 
> I have suspicion that this tile was around for a long time.
> So next step- removing built versions and stable versions?

Yes, we are working on that.

**Comment by Chezzo** [comment]
So what do I need to do to get my set back up?

**Comment by kevingranade** [comment]
I'd traced it back to MShock previously, but after some further digging, it looks like the source is BlockHead tileset, which was obsoleted a looong time ago.

That in turn seems to have gotten it from Raga tileset, which was never in the repository but was published on the forums:
https://discourse.cataclysmdda.org/t/raga-32x32-tileset-wip-85/4401

I would say at a minimum everything pulled in via Raga is suspect, and ideally we'd have a better idea of where *everything* comes from.

**Comment by lispcoc** [comment]
Why ChestHole16Tileset still exists?

**Comment by 18119928570** [comment]
Through it was a mistake,when it would come back in game again?It is quite a great tileset to us normal player

**Comment by ZhilkinSerg** [comment]
> 
> 
> Through it was a mistake,when it would come back in game again?It is quite a great tileset to us normal player

What was a mistake?

**Comment by kevingranade** [comment]
I think they mean, "a mistake happened including the bad tile(s)"

**Comment by terribleperson** [comment]
Since it's difficult verging on impossible to prove that an artist is the original creator of a tile, but easy to prove that someone is not, what's the official plan? All raga-derived tiles are suspect and no tileset can be included without those tiles being removed or replaced? All BlockHead-derived tiles are suspect? All MShock-derived tiles are suspect? I believe there is art in MShock32 that was originally created for Cataclysm, so I'd hate to see all those tiles have to be excised, especially considering

**Comment by Chezzo** [comment]
I hear you, man. That tileset represents MONTHS of pushing pixels around for me. 

I stuck up for you in my emails providing provenance.

**Comment by egomassive** [comment]
Waldo and Colleen's work should be safe too, but Waldo was not happy about being included in MShock's big merger. I mainly tried to imitate Waldo's style which I feel is significantly different from your style @Chezzo , so don't feel obligated to use my art. I would also trust Xotto's work since he started with high-quality assets based on Waldo's work. As for MShock's personal additions, he said his girlfriend helped him make them and he seems to have personal interest in them (i.e. he resubmitted some to SomeDeadGuy since they had degraded in quality.)

If Raga is the only perpetrator it's still a big problem. I looked through his set, and even if you exclude anything derived from Deon it still amounts to 2 to 3 hundred tainted tiles. Some have been replaced over the years but others have numerous derivatives. It'd be a huge job just to identify and blot out all the offenders in MShockXotto+, not to mention the juggernaut of MSX++DeadPeople. ChestHole sets would be the best recovery option since they are much less reliant on older tile sets.

**Comment by EvgenijM86** [comment]
Tiles are still available in git history. This means that you are still redistributing them with any git clone requests. During git revert 14e441003d479f5bbbd88f120b533b70b5b88bdb restores them.

**Comment by mryellow** [comment]
So what you're saying is I need to fork this repo and maintain the old history to ensure the game is playable?

**Comment by EvgenijM86** [comment]
I am not asking for anything. I am simply letting you know that this is still the case, because I got the impression that it was important for cata devs to not redistribute someone else assets without their permission. If its not important as long as it is "hidden under the rug" then I can absolutely understand that.

**Comment by kevingranade** [comment]
We've taken action to avoid distributing the works in question along with
the current game archive, which is the predominant method of distribution.
If the copyright holders were insistent that these files being available
from github presented a problem, we could look into expunging them from the
project history as well, but that would be a significantly disruptive
undertaking, so I would prefer not to do so.

**Comment by rightdroid** [comment]
> If the copyright holders were insistent that these files being available from github presented a problem, we could look into expunging them from the project history

Why is this not the default stance for the current build? Remove the offending artwork and leave the rest. I'm scratching my head trying to grasp how is it more reasonable to ask artists responsible for presumably thousands or tile images to prove these are original work instead of acting when someone objects (as, from the quoted part, seems to be the case for historic builds of the game). The whole burden of proof seems to be upside down here.

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/what-happened-to-old-tile-set-in-0-e/24498/5

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/what-happened-to-old-tile-set-in-0-e/24498/10
